src/main/resources/scipoolprod.properties

```diff
@@ -4,6 +4,7 @@ AWS_REGION=us-east-1
 SESSION_TIMEOUT_SECONDS=28800
 SESSION_NAME_CLAIMS=userid
 SESSION_TAG_CLAIMS=sub,userid,team,user_name,company,given_name,family_name
+REDIRECT_URIS=https://sc.sageit.org/synapse,https://connect.scipool.org/oauth2/idpresponse 
```

Comment:
The second URI should read `https://connect.scipoolprod.org/oauth2/idpresponse` I believe.

Reply:
Done!